traitsui/editor.py

```diff
@@ -129,6 +135,15 @@ class Editor(HasPrivateTraits):
     #: The current editor invalid state status:
     invalid = Bool(False)
 
+    #: A set to track values being updated to prevent infinite recursion.
```

Comment:
These are attributes that are already being used, but not set. Is that correct?

Reply:
This is the set of attributes on the object which have an active trait change in progress that is updating the editor.